update documentation (#19016)
arangodb/arangodb: Documentation/DocuBlocks/Rest/Queries/post_api_cursor_cursor_batch.md

successfully received and processed the batch so that the server can free up
resources.

> The time-to-live for the cursor will be renewed by this API call.

Reviewer: ```suggestion
The time-to-live for the cursor is renewed by this API call.
```
